config/babel/.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["es2015", "stage-1", "react"],
+  "presets": ["react", "react-native-stage-0"],
```

Comment:
Do all victory components need `stage-0` now for compatibility use with React Native? Or just some. If just some, we should have a conditional flag to switch which one to use (I can show you options as to how).